src/olympia/accounts/views.py

```diff
@@ -211,15 +217,18 @@ def inner(self, request):
 
 def add_api_token_to_response(response, user, set_cookie=True):
     # Generate API token and add it to the json response.
-    payload = jwt_api_settings.JWT_PAYLOAD_HANDLER(user)
-    token = jwt_api_settings.JWT_ENCODE_HANDLER(payload)
+    data = {
+        'secret': user.get_session_auth_hash(),
```

Comment:
Yeah, it will be in the base64 string. Couldn't figure out a good name, I didn't want to call it "session" something because it's not an actual session id, just an id that is *part* of the session.

Could call that `session_auth_hash` if we don't mind the longer string.